src/app/tests/suites/certification/Test_TC_ACFREMON_1_1.yaml

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) 2023 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 165.1.1. [TC-ACFREMON-1.1] Global Attributes with DUT as Server
+
+PICS:
+    - ACFREMON.S
+
+config:
+    nodeId: 0x12344321
+    cluster: "Activated Carbon Filter Monitoring"
+    endpoint: 1
+
+tests:
+    - label: "Wait for the commissioned device to be retrieved"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "Read the global attribute: ClusterRevision"
+      command: "readAttribute"
+      attribute: "ClusterRevision"
+      response:
+          value: 1
+          constraints:
+              type: int16u
+
+    - label: "Read the global attribute: FeatureMap"
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      PICS: "!ACFREMON.S.F00 && !ACFREMON.S.F01"
+      response:
+          value: 0
+          constraints:
+              type: bitmap32
+
+    - label:
+          "Given ACFREMON.S.F00(Condition) ensure featuremap has the correct bit
+          set"
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      PICS: ACFREMON.S.F00
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x1]
+
+    - label:
+          "Given ACFREMON.S.F01(Warning) ensure featuremap has the correct bit
+          set"
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      PICS: ACFREMON.S.F01
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x2]
+
+    - label: "Read the global attribute: AttributeList"
+      command: "readAttribute"
+      attribute: "AttributeList"
+      response:
+          constraints:
+              type: list
+              contains: [2, 65528, 65529, 65530, 65531, 65532, 65533]
+
+    - label:
+          "Read the feature dependent(ACFREMON.S.F00) attribute in AttributeList"
+      PICS: ACFREMON.S.F00
+      command: "readAttribute"
+      attribute: "AttributeList"
+      response:
+          constraints:
+              type: list
+              contains: [0, 1]
+
+    - label: "Read the global attribute: AcceptedCommandList"
```

Comment:
Should have a test that checks that attribute 3 is present if the right PICS is set.